diff --git a/go-controller/hack/build-go.sh b/go-controller/hack/build-go.sh
index bfcf0c8b..a068088e 100755
--- a/go-controller/hack/build-go.sh
+++ b/go-controller/hack/build-go.sh
@@ -23,7 +23,7 @@ build_binaries() {
     set -x
     for bin in "$@"; do
         binbase=$(basename ${bin})
-        go build -v \
+        CGO_ENABLED=0 go build -v \
             -mod vendor \
             -gcflags "${GCFLAGS}" \
             -ldflags "-B ${BUILDID} \
@@ -47,7 +47,7 @@ build_windows_binaries() {
     set -x
     for bin in "$@"; do
         binbase=$(basename ${bin})
-        GOOS=windows GOARCH=amd64 go build -v \
+        CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -v \
             -mod vendor \
             -gcflags "${GCFLAGS}" \
             -ldflags "-B ${BUILDID}" \